helm-charts/Makefile

```diff
@@ -17,13 +18,13 @@ helm_clean:
 
 helm_lint:
 	# Linting Helm Chart
-	helm lint --debug $(CHART_PATH)
+	$(HELM_VERSION) lint --debug $(CHART_PATH)
 
 helm_template:
-	./kafka-version-tpl.sh strimzi-kafka-operator/templates/_kafka_image_map.tpl
+	./kafka-version-tpl.sh $(HELM_VERSION)/strimzi-kafka-operator/templates/_kafka_image_map.tpl
 	# Generating rendered template files to: $(CHART_RENDERED_TEMPLATES_TMP)
 	mkdir -p $(CHART_RENDERED_TEMPLATES_TMP)
-	helm template --namespace myproject --output-dir $(CHART_RENDERED_TEMPLATES_TMP) $(CHART_PATH)
+	$(HELM_VERSION) template --namespace myproject --output-dir $(CHART_RENDERED_TEMPLATES_TMP) $(CHART_PATH)
 
 helm_install: helm_clean helm_template
```

Comment:
I think this will need some thoughts. Right now, the Helm2 chart holds the master copy to all installation files which are not CRD. So if you for example want to change something in the deployment (e.g. `install/cluster-operator/050-...yaml`), you edit it in the Helm Chart and run `make helm_install` whcih renders the regular YAML from the Helm Chart. Now the problem is, we cannot do this twice - once with Helm2 and once with Helm3. So I think we need to decide which one will be the master and generate the install files from it but also the other Helm Chart from it if possible.